plivo/utils/validators.py

```diff
@@ -180,6 +180,39 @@ def f(name, value):
     return required(f)
 
 
+def multiple_valid_integers():
+    def f(name, value):
+        if isinstance(value, int) and name == 'delay_dial':
```

Comment:
make this as 
`if isinstance(value, int) and name in ['delay_dial', 'ring_timeout']:` and remove the elif block below.